src/Assert.php

```diff
@@ -817,6 +818,24 @@ public static function uuid($value, $message = '')
         }
     }
 
+    public static function throws(callable $expression, $class, $message = '')
```

Comment:
sadly, I would typehint this only in phpdocs,
`callable` will not work on 5.3, while you could